docs/src/content/docs/introduction/http-signatures.mdx

```diff
@@ -41,20 +41,38 @@ The benefits of using Ed25519 include:
 
 For more information about the EdDSA and its variants, refer to <LinkOut href='https://datatracker.ietf.org/doc/html/rfc8032'>RFC8032</LinkOut>.
 
-{/* prettier-ignore */}
-{/* ### Message signature creation
+### Message signature creation
 
-:::caution
-This section is WIP
-:::
+The [RFC8032](https://www.rfc-editor.org/rfc/rfc8032) specification permits the use of different components of an HTTP message to generate a uniquely labelled message signature. There can be multiple signatures within each HTTP message, each one generated using a different signature algorithm.
```

Comment:
Should we link that spec or the [part about signatures in the GNAP spec](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol-16#section-7.3.1)?

Reply:
Yes, done, thank you!